content/blog/explore-the-hpe-greenlake-for-compute-ops-management-rest-api-using-curl-and-postman.md

```diff
@@ -0,0 +1,333 @@
+
+                 **   "hostname": "None"**
+
+The following cURL command includes the filter
+
+```shell
+curl -X GET “https://us-west2-api.compute.cloud.hpe.com/compute-ops/v1beta2/servers?filter=contains(hardware/model,'DL365')” -H "Authorization:Bearer <your access_token_here>"
+```
+
+![blog figure8](/img/greenlake-com-ops-api-curl5.png)
+
+Contains is another filter option to return resources where associated **hardware/model** contains **DL365.** The result of the command shows 4 server resources whose model name contains DL365.
+
+The filter query syntax supports a richer set of filters than the single operation in the previous example. Filtering syntax is broken down by Operations, Logic, and Types. In the previous example, [](<>)the operation was 'eq' for equality. Most comparison operations require the evaluated property name to the left of the operator and a literal to the right.
+
+![blog figure9](/img/greenlake-com-ops-api-curl16.png)
+
+To learn more about Filtering, see [the HPE GreenLake for Compute Ops Management guide.](https://developer.greenlake.hpe.com/docs/greenlake/services/compute-ops/public/guide/#filtering)
+
+The HPE GreenLake for Compute Ops Management API supports many resources that are described in the [API reference](https://developer.greenlake.hpe.com/docs/greenlake/services/compute-ops/public/openapi/compute-ops-latest/overview/) document, as illustrated below:
+
+![blog figure10](/img/greenlake-com-ops-api-curl17.png)
+
+Unique Resource Identifiers [(URIs)](<>)  are used to identify a resource. A URI is a full API path ending in an identification number. 
+
+For example:
+
+* compute-ops/v1beta2/servers/{serverId}
+* compute-ops/v1beta1/reports/{id}/data
+
+**v1beta1**, **v1beta2** in the URI is the version of the resource that is being accessed.
+
+One can invoke the common HTTP [](<>)methods, like GET, POST, PUT, PATCH, and DELETE, on resources in the HPE GreenLake for Compute Ops Management API as shown below for the **filters** resource:
+
+![blog figure11](/img/greenlake-com-ops-api-curl23.png)
+
+Refer to the [API reference](https://developer.greenlake.hpe.com/docs/greenlake/services/compute-ops/public/openapi/compute-ops-latest/overview/) site for a complete list of [](<>)methods supported by each API resource.
+
+# Using Postman
+
+Postman is a tool designed to build and use APIs.
+
+To get started, “create a request”, as shown below:
+
+![blog figure12](/img/greenlake-com-ops-api-curl7.png)
+
+In the request URL field, enter the endpoint URL: ﻿
+
+[](<>)**<https://us-west2-api.compute.cloud.hpe.com/compute-ops/v1beta2/servers?limit=2>**
+
+This is the base connectivity endpoint you have seen in earlier, followed by the resource URI you want to query ﻿**/compute-ops/v1beta2/servers** as described in the API reference table.
+
+![blog figure13](/img/greenlake-com-ops-api-curl25.png)
+
+In order to limit the output as documented in the request pane of the *servers* resource:
```

Comment:
In think lines 254 and 256 should be combined. I think you meant to say:
In order to limit the output as documented in the request pane of the servers resource, we use query parameter ?limit=2.
![image](https://user-images.githubusercontent.com/77496261/205347188-8a8e4b01-acb5-46ef-8ab8-c31556456d55.png)

Reply:
query parameter ?limit=2  is the parameter